BWDB-3718 SCL - Header bar

dlisnichenko-cv opened the pull request:
## PR Checklist

Check these before submitting your pull request:

- [x] Visual and behavioral components are separated
- [x] Exported components are documented with examples
- [x] Props have JSDoc comments
- [x] All relevant visual sub-components can be overridden via props

## New Visual Components
This pull request adds a header bar that may be reused in other places across Bandwidths' portals. 

There should be a new horizontal 'header bar' for use between the navigation menu and the main content on any given page. 

The header bar should be 50px tall
The header bar should have a bottom border.
style: 1px solid #c2c2c2;
![create](https://user-images.githubusercontent.com/35302487/44150878-88052438-a0a9-11e8-9380-25aadca73703.jpg)

dlisnichenko-cv commented:
Updated patch according to a new project structure

zelitzsi commented on packages/shared-components/src/components/HeaderBar/HeaderBar.js:
```diff
@@ -0,0 +1,11 @@
+
+const HeaderBar = styled.div.withConfig({ displayName: 'HeaderBar' })`
+  height: 50px;
+  border-bottom: 1px solid #c2c2c2;
```
Please use themeGet to use the theme value for this color and line thickness.

dlisnichenko-cv replied:
ok, done

dlisnichenko-cv commented:
Fixed review comments 
![scr1](https://user-images.githubusercontent.com/35302487/44351401-73dc1980-a4aa-11e8-8125-f69cdec1621d.png)

zelitzsi commented on packages/shared-components/src/components/HeaderBar/HeaderBar.js:
```diff
@@ -1,8 +1,11 @@
-  border-bottom: 1px solid #c2c2c2;
+  display: flex;
+  align-items: center;
+  border-bottom: ${irisTheme.thicknesses.normal} solid ${irisTheme.colors.gray.border};
```
Use `themeGet()`; it will adapt to the current theme rather than assuming the current theme is IRIS (a relatively safe assumption for now, but not future proof).

dlisnichenko-cv replied:
ThemeGet is marked as deprecated. Ok, will switch to it

zelitzsi replied:
Yeah. For some history, we deprecated themeGet because we wanted to switch to using css vars. However, we are still determining how we are supporting IE, so we should continue to use it for the meantime. The alternative would be to use css vars in styled components directly.

dlisnichenko-cv commented:
Switched to themeGet and added padding
![scr1](https://user-images.githubusercontent.com/35302487/44352340-c585a380-a4ac-11e8-9053-2334b772dd52.png)
